packages/routes/test/RouteRegistration.tsx

```diff
@@ -41,7 +41,9 @@ export class Test extends RouteTest {
 
     const request = httpMocks.createRequest({ method: 'GET', url: '/api' })
     const response = httpMocks.createResponse()
-    router.handle(request, response, undefined)
+
+    router.handle(request, response)
+    await new Promise(resolve => setTimeout(resolve, 0)) // TODO: find a less dirty way to wait for async execution
```

Comment:
there probably is some way to write this in a better way, but nothing comes to my mind yet. I will accept the pull request, but write this one down for later discussion within the team. :-)